setup.py

```diff
@@ -79,14 +79,15 @@ def get_long_description() -> str:
         "regex>=2020.1.8",
         "pathspec>=0.8.1, <1",
         "dataclasses>=0.6; python_version < '3.7'",
-        "typing_extensions>=3.7.4; python_version < '3.8'",
+        "typing_extensions>=3.7.4; python_version < '3.10'",
         "mypy_extensions>=0.4.3",
     ],
     extras_require={
         "d": ["aiohttp>=3.6.0", "aiohttp-cors>=0.4.0"],
         "colorama": ["colorama>=0.4.3"],
         "python2": ["typed-ast>=1.4.2"],
         "uvloop": ["uvloop>=0.15.2"],
+        "jupyter": ["ipython>=7.8.0", "tokenize-rt>=3.2.0"],
```

Comment:
Since I see we're using dataclasses in black code now, should we explicitly add it as a dep for when we're < 3.7 ?